doc/changes.md

```diff
@@ -1,5 +1,92 @@
 # Changelog
 
+## Release v0.6.0 (20YY/MM/DD)
```

Comment:
Putting down Monday here (UTC time) just in case, but I'm hoping to get this over by Sunday in the US if possible.

```suggestion
## Release v0.6.0 (2022/03/14)
```

Reply:
I think we should edit https://github.com/GenericMappingTools/pygmt/issues/1801 to 03/14 to give opportunities for comments at https://github.com/GenericMappingTools/pygmt/pull/731 and https://github.com/GenericMappingTools/pygmt/pull/1812. It would've been nice to wrap this up today (really missing that extra hour), but a PI day release could also be fun.

Reply:
Oo yes, definitely do 3/14! I just realized we'll have 3 new functions (`grdhisteq`, `triangulate`, `pygmt.datasets.load_sample_data`) and 14 inline examples (if we merge in just one of #1719 or #1729) :laughing:

Reply:
The plot is really nice. Do you mind sharing your code? I'd be interested in trying a couple adjustments.

Reply:
Sure, I was thinking of putting it in the forum post but couldn't find a good place to put it. The code is:

```python
import pygmt

fig = pygmt.Figure()
fig.basemap(region=[0, 8, 0, 3], projection="X12c/4c", frame=True)

pen = "1.5p"
fig.plot(x=1, y=1.5, style="l3.5c+t3", color="dodgerblue3", pen=pen)
fig.plot(x=2, y=1, style="l3.5c+t.", color="red3", pen=pen)
fig.plot(x=3, y=1.5, style="l3.5c+t1+fCourier-Bold", color="seagreen", pen=pen)
fig.plot(x=4.3, y=1.5, style="l3.5c+t4+fTimes-Italic", color="gold", pen=pen)
fig.plot(x=5.6, y=1.5, style="l3.5c+t1+fBookman-Light", color="pink", pen=pen)
fig.plot(x=7, y=1.5, style="l3.5c+t\160+fSymbol,darkorange", color="magenta4", pen=pen)

fig.show()
```